docs/user/cassandra_datastore.rst

```diff
@@ -46,15 +46,15 @@ installation directory).
 
 After you've successfully installed and configured Cassandra, you can
 move on to installing the GeoMesa Cassandra distribution.
-Start by choosing or creating a directory where you want
-to put it. Then ``cd`` into that directory and type::
+To start, go to
+https://repo.locationtech.org/content/repositories/geomesa-snapshots/org/locationtech/geomesa/geomesa-cassandra-dist_2.11/1.3.0-m3-SNAPSHOT/ ,
```

Comment:
we should probably keep it pointing to the release repo - we're going through and getting everything in order for 1.3.0 right now, so we'd just be reverting it again in a day or two. will the existing docs be correct once we have a 1.3.0 release?